tests/model/database/test_global_sqlite.py

```diff
@@ -0,0 +1,205 @@
+import asyncio
+import unittest
+
+from gi.repository import GLib
+from gi.repository import Gio
+from gi.repository.GLib import DateTime, TimeZone
+from gi.events import GLibEventLoopPolicy
+
+from sbaid.common.vehicle_type import VehicleType
+from sbaid.model.database.foreign_key_error import ForeignKeyError
+from sbaid.common.a_display import ADisplay
+from sbaid.common.b_display import BDisplay
+from sbaid.common.simulator_type import SimulatorType
+from sbaid.model.database.global_sqlite import GlobalSQLite
+
+class GlobalSQLiteTest(unittest.TestCase):
+
+    def setUp(self):
+        asyncio.set_event_loop_policy(GLibEventLoopPolicy())
+        loop = asyncio.get_event_loop()
+        task = loop.create_task(GlobalSQLiteTest().test())
+        loop.run_until_complete(task)
+
+    async def test(self) -> None:
+        await self.remove()
+        await self.result()
+        await self.snapshot()
+        await self.cross_section_snapshot()
+        await self.lane_snapshot()
+        await self.times()
+        await self.tags()
+        await self.foreign_key_error()
+
+    def on_delete_async(feld, file, result, user_data):
+        try:
+            file.delete_finish(result)
+        except GLib.Error as e:
+            raise e
```

Comment:
warum wrapst du das in try wenn du das gleiche wieder raisest? ist das convention?